STX.REST.RESTFulSense.Clients.Tests.Unit/Services/Foundations/HttpExchanges/HttpExchangeServiceTests.Logic.Get.cs

```diff
@@ -0,0 +1,71 @@
+// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System.Net.Http;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using STX.REST.RESTFulSense.Clients.Models.Services.HttpExchanges;
+using Xunit;
+
+namespace STX.REST.RESTFulSense.Clients.Tests.Unit.Services.Foundations.HttpExchanges
+{
+    public partial class HttpExchangeServiceTests
+    {
+        [Fact]
+        private async Task ShouldSendHttpRequestWhenGetAsyncIsCalledAsync()
+        {
+            // given
+            dynamic randomProperties = CreateRandomExchangeProperties();
+
+            HttpExchange inputHttpExchange =
+                CreateHttpExchangeRequest(randomProperties);
+
+            HttpExchange expectedHttpExchage =
+                CreateHttpExchangeResponse(
+                    inputHttpExchange,
+                    randomProperties);
+
+            HttpRequestMessage inputHttpRequestMessage =
+                CreateExchangeRequestMessage(randomProperties);
+
+            HttpResponseMessage expectedHttpResponseMessage =
+                CreateExchangeResponseMessage(randomProperties);
+
+            this.httpBroker.Setup(broker =>
+                broker.SendRequestAsync(
+                    It.IsAny<HttpRequestMessage>(), default))
+                        .ReturnsAsync(expectedHttpResponseMessage);
+
+            // when
+            HttpExchange actualHttpExchange =
+                await this.httpExchangeService.GetAsync(
+                    inputHttpExchange);
+
+            // then
+            actualHttpExchange.Should().BeEquivalentTo(
+                expectedHttpExchage,
+                options => options.Excluding(
+                    x => x.Response.StreamContent));
```

Comment:
Don't use x. It is not descriptive

```suggestion
                    httpExchange => httpExchange.Response.StreamContent));
```